MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -2039,7 +2050,8 @@
 		// XXX We really want to use the device's Metal features instead of duplicating the
 		// logic from MVKPhysicalDevice, but those may not have been initialized yet.
 #if MVK_MACOS
-		 (isStencilFormat(vkDesc.mtlPixelFormat) && (!_physicalDevice || ![mtlDev supportsFeatureSet: MTLFeatureSet_macOS_GPUFamily2_v1]))
+		(isStencilFormat(vkDesc.mtlPixelFormat) && (!_physicalDevice || !([mtlDev supportsFamily: MTLGPUFamilyMac2] ||
```

Comment:
This won't work on Mojave. It doesn't yet have `-[MTLDevice supportsFamily:]`.

Reply:
Good catch. I've refactored this section. Please review.

Reply:
Looks a lot cleaner. Thanks.